### PR TITLE
fix: mentionTemplate should be passed when mapped to sendMessage

### PR DESCRIPTION
**Pull request by bang9**
## Description
`mentionedMessageTemplate ` should be passed to `sendMessage`

ticket: [CLNP-2261]

[CLNP-2261]: https://sendbird.atlassian.net/browse/CLNP-2261?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/modules/Channel/components/MessageInputWrapper/index.tsx [hunk 1]
```diff
@@ -37,22 +36,27 @@ export const MessageInputWrapper = (props: MessageInputWrapperProps) => {
...
-      )}
-      sendMultipleFilesMessage={({ fileInfoList, parentMessageId }) => (
-        sendMultipleFilesMessage(fileInfoList.map((fileInfo) => fileInfo.file) as File[], { parentMessageId } as SendableMessageType)
-      )}
+      sendUserMessage={(params) => {
+        return sendMessage({
+          message: params.message,
+          mentionTemplate: params.mentionedMessageTemplate,
+          mentionedUsers: params.mentionedUsers,
+          quoteMessage: context.quoteMessage,
```
Comment (HoonBaek) [inline src/modules/Channel/components/MessageInputWrapper/index.tsx:48]: Why don't we get `quoteMessage` from context using an object destructuring assignment?

```tsx
const {
  ...,
  quoteMessage,
} = context;
```

Comment (bang9) [reply src/modules/Channel/components/MessageInputWrapper/index.tsx:48]: There's no particular reason! 😄

Comment (bang9) [reply src/modules/Channel/components/MessageInputWrapper/index.tsx]: addressed in https://github.com/sendbird/sendbird-uikit-react/pull/971/commits/da2a4d3caca8a42c3b4a72e58f35d7dbd061c697

**Review by HoonBaek — APPROVED** [review]
LGTM

**Pull request merged**